### ISSUE TITLE
SUB - 1.4.4 - Resize Text - Define "loss of content or functionality"

### ISSUE THREAD

**Issue by TanyaVanWorkum** [w3c/matf#137]

**Comment by JJdeGroot** [comment]
Out of scope, if definition is needed it should be provided by WCAG, not by MATF
